go.mod

```diff
@@ -29,6 +29,10 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.20.2
 	k8s.io/mount-utils => k8s.io/mount-utils v0.20.2
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.2
+	sigs.k8s.io/kustomize/api => sigs.k8s.io/kustomize/api v0.6.9
```

Comment:
This needs to be pinned in `require`. Replace is actuallt a hack for k8s because they don't pin their dependency in `go.mod`